decko/lib/decko/config/environments/cucumber.rb

```diff
@@ -1,5 +1,5 @@
 # -*- encoding : utf-8 -*-
-Decko.application.class.configure do
+Decko::Application.configure do
```

Comment:
This is wrong. 

When you make a deck, eg wikirate, it generates a new application class, eg WikirateApplication, which inherits from Decko::Application.  Decko.application.class returns WikirateApplication.